### ISSUE TITLE
vdbbin gui flakiness

### ISSUE THREAD

**Issue by atlas0fd00m** [vivisect/vivisect#281]
i'm guessing that since the pyqt5 upgrade, we haven't caught all the possible exceptions, causing the GUI to crash when it should spit out an exception message.

need to clean up vdb gui exception handling code (or rather, add some in, since pyqt5 removed the backstop and allows proggies to crash easily.

**Comment by rakuy0** [comment]
I agree, and do you have some of the exceptions you're seeing on hand? Good place to start tracking those down.

**Comment by rakuy0** [comment]
bump
